.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto
```

Comment:
I don't think this is needed on this PR. It might be appropriate for a *different* PR about improving QoL, but not for a refactor. File should be removed.